config/flags.go

```diff
@@ -77,4 +77,16 @@ var (
 		Name:  "maxnetdelay",
 		Usage: "Max network delay for broadcasting",
 	}
+	FastSyncFlag = cli.BoolFlag{
+		Name:  "fast",
+		Usage: "Enable fast sync",
+	}
+	SnapshotRangeFlag = cli.Uint64Flag{
```

Comment:
Do we really need this in CLI params? nodes cannot have different number here (except for testing purpose, but you can set it through `config.json`)